cpp/Ice/greeter/README.md

```diff
@@ -1,14 +1,25 @@
 # Greeter
 
-The Greeter demo illustrates how to send a request and wait for the response.
+The Greeter demo illustrates how to build a simple application with Ice.
 
-To build the demo run:
+The client demonstrates the 3 APIs you can use to make remove invocations using generated Ice proxies:
```

Comment:
There's a typo, and saying "the client" sounds weird and ambiguous. We were just talking about 'the greeter' and didn't mention anything about a client until now.
```suggestion
It features a client which demonstrates the 3 APIs you can use to make remote invocations using generated Ice proxies:
```